pocolog2msgpack.py

```diff
@@ -49,7 +49,7 @@ def _extract_keys(sample, whitelist=(), keys=()):
         for k in sample.keys():
             result.extend(_extract_keys(sample[k], whitelist, keys + (k,)))
         return result
-    elif isinstance(sample, list) and (".".join(keys) in whitelist):
+    elif isinstance(sample, list) and (".".join([str(key) for key in keys]) in whitelist):
```

Comment:
I suppose `map(str, keys)` could be faster since it is implicit. What do you think?

Reply:
You're right and it fits more with the rest of the code.